### PR TITLE
(hub-test): implemented DiscountedBalances unit test coverage

### PR DESCRIPTION
**Pull request by roleengineer**

### CODE REVIEW

**Comment by roleengineer** [comment]
solving this https://github.com/aboutcircles/circles-contracts-v2/issues/99

**Comment by benjaminbollen** [comment]
congrats on 100th number 🥳

File: test/circles/MockDiscountedBalances.sol [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    constructor(uint256 _inflationDayZero) DiscountedBalances(_inflationDayZero) {}
+
+    // External functions
+
+    function maxBalance() external pure returns (uint256) {
+        return MAX_VALUE;
+    }
+
+    function getAvatarBalanceValue(uint256 id, address avatar) external view returns (uint192) {
```
Comment (benjaminbollen) [inline test/circles/MockDiscountedBalances.sol:21]: on the `candidate/stable` branch we should look to make these struct readable, among others

**Review by benjaminbollen — APPROVED** [review]
Looks very good to me! We're officially started on this epic 🏇

**Pull request merged**